qemu/tests/cfg/thin_provision_guest_fstrim.cfg

```diff
@@ -0,0 +1,43 @@
+- thin_provision_guest_fstrim:
+    only Linux
+    only virtio_scsi
+    type = thin_provision_guest_fstrim
+    start_vm = no
+    not_preprocess = yes
+    kill_vm = yes
+    data_tag = "stg1"
+    images += " ${data_tag}"
+    drv_extra_params_stg1 += "discard=unmap"
+    image_format_stg1 = raw
+    image_raw_device_stg1 = yes
+    force_create_image_stg1 = no
+    drive_format_stg1 = scsi-block
+    remove_image_stg1 = no
+    disk_size = 256
+    pre_command = "modprobe -r scsi_debug; modprobe scsi_debug dev_size_mb=${disk_size} lbpu=1 lbpws=1 lbprz=0"
+    post_command = "modprobe -r scsi_debug"
+#   serial port setting
+    serials += " org.qemu.guest_agent.0"
+    serial_type_org.qemu.guest_agent.0 = virtserialport
+    guest_agent_name = "org.qemu.guest_agent.0"
+    guest_agent_serial_type = virtio
+    variants:
+        - ext4:
+            format = "ext4"
+        - xfs:
+            format = "xfs"
+    variants:
+        - scsi_block:
+            drive_format_stg1 = scsi-block
+        - scsi_hd:
+            drive_format_stg1 = scsi-hd
+            disk_serial = TARGET_DISK1
+            blk_extra_params_stg1 += "serial=${disk_serial}"
+#    test command in guest
+    guest_mount_point = "/home/test"
+    guest_test_file = "${guest_mount_point}/test.img"
+    guest_format_command = "mkdir -p ${guest_mount_point};mkfs.${format} {0} && mount {0} ${guest_mount_point}"
+    guest_dd_command = "dd if=/dev/zero of=${guest_test_file}"
+    guest_rm_command = "rm -rf ${guest_test_file};sync"
+
+
```

Comment:
Please only one new blank line at the end of the file.

Reply:
updated